tutorial/EASIER-net Tutorial.ipynb

```diff
@@ -97,6 +123,111 @@
     "mse = np.mean(np.power(predictions - test_y, 2))\n",
     "print(mse)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6ee647bc",
+   "metadata": {},
+   "source": [
+    "##### Classification"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "72c99ff0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def make_class_data(n_samples, num_classes, n_features=20): \n",
+    "    x = np.random.rand(n_samples,n_features)\n",
+    "    y = np.random.randint(low=0, high=num_classes, size=n_samples, dtype=int)\n",
+    "    return x,y\n",
+    "\n",
+    "np.random.seed(0)\n",
+    "train_x, train_y = make_class_data(1000, 2)\n",
+    "test_x, test_y = make_class_data(1000, 2)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "dc77b3ed",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Train an easier net\n",
+    "e_net = EasierNetEstimator(\n",
+    "    n_estimators=2,\n",
+    "    input_filter_layer=True,\n",
+    "    n_layers=3,\n",
+    "    n_hidden=20,\n",
+    "    full_tree_pen=0.1,\n",
+    "    input_pen=0.01,\n",
+    "    batch_size=100,\n",
+    "    num_classes=2, #set this to however many class you have in your dataset\n",
+    "    weight=[],\n",
```

Comment:
I'd set this to `[1,1]` to be explicit.

Reply:
should I set it to [1,1] for regression as well?